backend/src/packages/chats/chat.controller.ts

```diff
@@ -254,6 +254,7 @@ class ChatController extends BaseController {
     }>,
   ): Promise<APIHandlerResponse> {
     const chatMessageToCreateData: ChatMessageCreatePayload = {
+      isGeneratedByChatbot: options.body.isGeneratedByChatbot,
```

Comment:
we don't need this property, we need to send request to create a message. After it finishes send request to generate a reply. To detect whether it's a current user message, we can compare sender id and authenticated user id
```suggestion
```